app/models/concerns/request_helper.rb

```diff
@@ -135,29 +135,36 @@ def get_proxy(uri, format = :array, force = false)
 
     def request_url(url, verb = 'Get')
       uri = self.parse_url(url)
-      self.request_uri(uri, verb)
+      begin
+        self.request_uri(uri, verb)
+      rescue Net::HTTPClientException
+        self.request_uri(uri, verb, skip_proxy = true)
+      end
     end
 
-    def request_uri(uri, verb = 'Get')
-      http = Net::HTTP.new(uri.host, uri.inferred_port)
-      http.read_timeout = PenderConfig.get('timeout', 30).to_i
-      http.use_ssl = uri.scheme == 'https'.freeze
+    def request_uri(uri, verb = 'Get', skip_proxy = false)
+      http = self.initialize_http(uri, skip_proxy)
       headers = {
         'User-Agent' => self.html_options(uri)['User-Agent'],
         'Accept-Language' => LANG,
       }.merge(self.get_cf_credentials(uri))
 
       request = "Net::HTTP::#{verb}".constantize.new(uri.to_s, headers)
       request['Cookie'] = self.set_cookies(uri)
+
+      http.request(request)
+    end
+
+    def initialize_http(uri, skip_proxy = false)
+      http = Net::HTTP.new(uri.host, uri.inferred_port)
       proxy_config = self.get_proxy(uri, :hash)
-      if proxy_config
-        proxy = Net::HTTP::Proxy(proxy_config['host'], proxy_config['port'], proxy_config['user'], proxy_config['pass'])
-        proxy.start(uri.host, uri.inferred_port, use_ssl: uri.scheme == 'https') do |http2|
-          http2.request(request)
-        end
-      else
-        http.request(request)
+      if proxy_config && skip_proxy == false
```

Comment:
Minor style suggestion: For boolean variables, it's more conventional to use boolean operators, e.g., `!skip_proxy` instead of `skip_proxy == false` (another bonus is that the boolean operator can match other falsy values beyond only `false`).